CoreJs-part3.js

```diff
@@ -0,0 +1,111 @@
+//CoreJS. Part 3.
+//1. Write a JavaScript function that takes an array of numbers as input and returns a new array where each element is multiplied by 2.
+function byTwo(numbers) {
+  return numbers.map((number) => number * 2);
+}
+
+const values = [1, 2, 3, 4, 5];
+console.log(
+  " - Task1. new array [1, 2, 3, 4, 5] multiply by2 = " + byTwo(values)
+);
+
+//================================================================================================
+//2. Write a JavaScript function that takes an array as input and returns the length of the array.
+function arrayLength(numbers) {
+  return numbers.reduce((accumulator) => accumulator + 1);
+}
+
+console.log(
+  ` - Task2. array [1, 2, 3, 4, 5] length custom function = ${arrayLength(
+    values
+  )} and predifined = ${values.length}`
+);
+
+//==========================================================================================================================================
+//3. Write a JavaScript function that takes an array of strings as input, sorts the strings in ascending order, and returns the sorted array.
+function sortArray(values) {
+  return values.sort();
+}
+const stringValues = ["ee", "aa", "kk", "bb", "cc", "dd", "cd"];
```

Comment:
Please try to run the code below. I think that you can optimize it a bit.

`sortArray("Ee", "aa", "kk", "bb", "cc", "dd", "cd"])`

'Ee' will be first item in the array.
The reason why it's like that is because of ASCII. **A,B,C** and so on has lower ASCII Code then **a,b,c** 

http://sticksandstones.kstrom.com/appen.html

Reply:
I modified to make it case insensitive, yet according to machine logic - putting Ee as first element is logical XD